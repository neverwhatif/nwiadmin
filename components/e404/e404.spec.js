import React from 'react';
import { shallow } from 'enzyme';

import E404 from './index';

describe('E404', () => {
    const component = shallow(<E404 />);

    it('should render', () => {
        expect(component).toMatchSnapshot();
    });
    it('should render text', () => {
        expect(component.text()).toBe('E404');
    });
});
