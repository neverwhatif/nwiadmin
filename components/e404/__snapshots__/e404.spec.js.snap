// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`E404 should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <E404 />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "E404",
    },
    "ref": null,
    "rendered": "E404",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "E404",
      },
      "ref": null,
      "rendered": "E404",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
